CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog], and this project adheres to
+[Semantic Versioning]. Numbers in parentheses are related issues or pull
+requests.
+
+## [Unreleased]
+
+### Added
+
+- Read allowed file formats from a CSV file (#60)
+
+## [0.1.0] - 2024-09-19
```

Comment:
Right now the version in VERSION.txt is `0.0.0`, but I thought it would be better to create the tag as `v0.1.0` and bump VERSION.txt to `0.2.0` directly in the next release.